src/components/ModelNavigator/DataDictionary/Table/DataDictionaryPropertyTable/component/tableHead.js

```diff
@@ -10,6 +10,9 @@ const TableHeader = ({ classes, hideIsRequired }) => (
       <th className={classes.item}>
         Type
       </th>
+      <th>
```

Comment:
Could we also add the `className={classes.item}` prop, this header has slightly different styling than the rest.